src/components/Select/Select.tsx

```diff
@@ -15,13 +16,17 @@ import SelectMenu from './components/SelectMenu/SelectMenu';
 import { rightIconContainer, selectWrapper } from './Select.style';
 import Loader from 'components/Loader';
 
-export type SelectOption = {
```

Comment:
I changed this part so we can have only one reference of what an Option should be like. So this will be used by anyone else